python/ql/src/Security/CWE-209/StackTraceExposure.ql

```diff
@@ -19,6 +19,5 @@ import DataFlow::PathGraph
 
 from Configuration config, DataFlow::PathNode source, DataFlow::PathNode sink
 where config.hasFlowPath(source, sink)
-select sink.getNode(), source, sink,
-  "$@ flows to this location and may be exposed to an external user.", source.getNode(),
-  "Stack trace information"
+select sink.getNode(), source, sink, "This information exposed to the user depends on $@.",
+  source.getNode(), "stack trace information"
```

Comment:
The message sounds strange. Personally, I feel this fits the ``or alternatively (if the first option doesn't work) `[User-provided value] flows to this location and is used in a path`.`` part of the [style guide](https://github.com/github/codeql/blob/main/docs/query-metadata-style-guide.md#alert-messages).

Can we please keep the old message?